ddtrace/appsec/_iast/_handlers.py

```diff
@@ -403,3 +405,65 @@ def _on_set_request_tags_iast(request, span, flask_config):
             OriginType.PARAMETER,
             override_pyobject_tainted=True,
         )
+
+
+def _on_django_finalize_response_pre(ctx, after_request_tags, request, response):
+    from .._asm_request_context import is_stacktrace_reported
+
+    if is_stacktrace_reported() or not _is_iast_enabled() or not is_iast_request_enabled() or not response:
+        return
+
+    try:
+        content = response.content.decode("utf-8")
```

Comment:
I wonder how safe it is to assume the response will always be "utf-8"

Reply:
Only in 99% of the cases according to wikipedia, otherwise, well, it will fall in the Except and we won't report it.

Reply:
Maybe worth noting that we can check Django settings for DEFAULT_CHARSET:
https://docs.djangoproject.com/en/5.1/ref/request-response/#id4

Reply:
PS: Since the stacktraces are almost warranteed to be UTF-8 and we don't care about the rest, I added `ignore=True`. Thanks!